src/modules/nameSchema/assets/helper/numeral.ts

```diff
@@ -0,0 +1,216 @@
+    let str = '';
+    let rest = num;
+
+    do {
+        const q = rest % 26;
+        rest = Math.floor(rest / 26);
+        str = String.fromCharCode(64 + q) + str;
+    } while (rest > 0);
+
+    return str;
+};
+
+/**
+ * Convert a number to ICAO alphabet word.
+ * @param num - Number to convert.
+ * @returns ICAO alphabet word.
+ * @see https://de.wikipedia.org/wiki/ICAO-Alphabet
+ */
+export const convertNumberToICAOAlpha = (num: number): string => {
+    const alphabet = [
+        'Alfa',
+        'Bravo',
+        'Charlie',
+        'Delta',
+        'Echo',
+        'Foxtrot',
+        'Golf',
+        'Hotel',
+        'India',
+        'Juliett',
+        'Kilo',
+        'Lima',
+        'Mike',
+        'November',
+        'Oscar',
+        'Papa',
+        'Quebec',
+        'Romeo',
+        'Sierra',
+        'Tango',
+        'Uniform',
+        'Victor',
+        'Whiskey',
+        'X-Ray',
+        'Yankee',
+        'Zulu',
+    ];
+    return alphabet[(num - 1) % alphabet.length];
+};
+
+/**
+ * Convert a number to a greek letter.
+ * @param num - Number to convert.
+ * @returns Greek letter.
+ */
+export const convertNumberToGreek = (num: number): string => {
+    const alphabet = [
+        'Alpha',
+        'Beta',
+        'Gamma',
+        'Delta',
+        'Epsilon',
+        'Zeta',
+        'Eta',
+        'Theta',
+        'Iota',
+        'Kappa',
+        'Lambda',
+        'My',
+        'Ny',
+        'Xi',
+        'Omikron',
+        'Pi',
+        'Rho',
+        'Sigma',
+        'Tau',
+        'Ypsilon',
+        'Phi',
+        'Chi',
+        'Psi',
+        'Omega',
+    ];
+    return alphabet[(num - 1) % alphabet.length];
+};
+
+/**
+ * Convert a string number to a string of emoji digits.
+ * @param string - String number to convert.
+ * @returns String of emoji digits.
+ */
+export const convertStringNumberToEmoji = (string: string): string => {
+    const emojiNum = (digit: number) =>
+        `${String.fromCodePoint(48 + (digit % 10))}\ufe0f\u20e3`;
+
+    let str = '';
+    for (const element of string) str += emojiNum(Number(element));
+
+    return str;
+};
+
+/**
+ * Convert a number to a german phonetic alphabet word.
+ * @param num - Number to convert.
+ * @returns German phonetic alphabet word.
+ * @see https://de.wikipedia.org/wiki/Buchstabiertafel
+ */
+export const convertNumberToGermanPhonetic = (num: number): string => {
+    const alphabet = [
+        'Anton',
+        'Berta',
+        'Cäsar',
+        'Dora',
+        'Emil',
+        'Friedrich',
+        'Gustav',
+        'Heinrich',
+        'Ida',
+        'Julius',
+        'Kaufmann',
+        'Ludwig',
+        'Martha',
+        'Nordpol',
+        'Otto',
+        'Paula',
+        'Quelle',
+        'Richard',
+        'Samuel',
+        'Theodor',
+        'Ulrich',
+        'Viktor',
+        'Wilhelm',
+        'Xanthippe',
+        'Ypsilon',
+        'Zacharias',
```

Comment:
I think we should stick to the most up to date DIN norm for the German phonetic alphabet (DIN 5009).
A table can be found [here](https://www.buchstabieralphabet.org/)

Reply:
As far as I'm concernend there is no county, where the new alphabet is in use. Since Hamburg uses the old one I vote against the replacement. Maybe as addition.

BTW: https://de.wikipedia.org/wiki/Buchstabiertafel?wprov=sfla1 has german old, german new, austrian and icao alphabet side by side ;)

Reply:
We would stick to something that is over 70 years old. I don't think that we as modern and young developers should implement something that is no longer the standard.

Reply:
B - 🥖

I like the idea of having the option to choose between phonetic alphabets.